connector/controllers/main.py

```diff
@@ -112,11 +112,10 @@ def retry_postpone(job, message, seconds=None):
             retry_postpone(job, unicode(err), seconds=err.seconds)
             _logger.debug('%s postponed', job)
 
-        except (FailedJobError, Exception):
+        except:
```

Comment:
I was also wondering why the `FailedJobError` is there. This code comes from https://github.com/OCA/connector/blob/1dbde5d23dbd97c5fee4accd408cb619bfd36ce8/connector/queue/worker.py#L150 (@guewen's). 

Still, the current except statement is equivalent to `except Exception` which catches everything for all practical purposes (https://docs.python.org/2/library/exceptions.html#exceptions.Exception).

@gfcapalbo can you say more about your problem?